flowmachine/flowmachine/core/mixins/exposed_datetime_mixin.py

```diff
@@ -0,0 +1,34 @@
+from datetime import date, datetime
+
+
+class ExposedDatetimeMixin:
```

Comment:
Can this go to a separate PR and be accompanied by tests please? See also: https://github.com/Flowminder/FlowKit/blob/008d7f4e9c720629d3b3d45db40c200899078b05/flowmachine/flowmachine/utils.py#L57 which harmonises date inputs to `"YYYY-MM-DD HH:MM:SS"` form.